core/types/calls.cc

```diff
@@ -563,7 +563,7 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
     auto errLoc = (funLoc.exists() && !funLoc.empty()) ? funLoc : args.callLoc();
     if (symbol == core::Symbols::untyped()) {
         auto what = core::errors::Infer::errorClassForUntyped(gs, args.locs.file, args.thisType);
-        if (auto e = gs.beginError(args.receiverLoc(), what)) {
+        if (auto e = gs.beginError(errLoc, what)) {
```

Comment:
This now uses the same location that we report "this method is missing" errors on, which seems fine. In a sense, the method "is" missing (because the call was on untyped, so Sorbet couldn't find it).

Reply:
A counterpoint: I think this is the first time we're reporting a usage of untyped error where the highlighted region is not a complete expression.

We weren't using that invariant before, but I wanted to call out that we're now losing it.